packages/admin-panel/src/rootReducer.js

```diff
@@ -2,23 +2,22 @@
  * Tupaia MediTrak
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
-
 import { combineReducers } from 'redux';
 import { reducer as authentication, LOGOUT } from './authentication';
 import { reducer as tables } from './table';
 import { reducer as autocomplete } from './autocomplete';
 import { reducer as editor } from './editor';
 import { reducer as dataChangeListener } from './dataChangeListener';
 
-const appReducer = combineReducers({
-  authentication,
-  tables,
-  autocomplete,
-  editor,
-  dataChangeListener,
-});
-
 export const rootReducer = (state, action) => {
+  const appReducer = combineReducers({
```

Comment:
I had to move the combineReducers inside the root reducer as I noticed a race condition where the sub-reducers hadn't finished runnning when the rootReducer ran